docs/general/ddl/data-types.rst

```diff
@@ -962,41 +962,92 @@ subcolumns. One can retrieve them, sort by them and use them in where clauses.
 ``ignored``
 -----------
 
-The third option is ``ignored`` which results in an object that allows
-inserting new subcolumns but this adding will not affect the schema, they are
-not mapped according to their type, which is therefor not guessed as well. You
-can in fact add any value to an added column of the same name. The first value
-added does not determine what you can add further, like with ``dynamic``
-objects.
-
-An object configured like this will simply accept and return the columns
-inserted into it, but otherwise ignore them.
-
+The third option is ``ignored``. Explicitly defined columns within an
+``ignored`` object behave the same as those within object columns declared as
+``dynamic`` or ``strict``. The columns are indexed unless indexing is explicitly
+turned off using ``INDEX OFF`` and values inserted are validated. The difference
```

Comment:
can you clarify what "and values inserted are validated" means? the passive voice makes it a little ambiguous. a switch to active voice would probably clear it up

is the user expected to validate values when inserting, or does CrateDB validate values when the user tries to insert them? (another way of putting it: is the validation a part of a "columns are indexed unless X and Y" precondition or is validation dependant upon indexing?)

Reply:
CrateDB validates the values according to the defined schema.  So the type must fit, and if there are further constraints, like `CHECK` the values must pass these checks.

Not sure how to phrase that better in a succinct way.

Reply:
Sorry but I don't get what can be ambiguous with "values are validated".
If a user should validate it, then it wouldn't be "**are** validated".

Could you please make a suggestion? 

Would using `... and column constraints and data type restrictions are ensured` instead help?

Reply:
@seut the sentence can be read two ways:

1. "The columns are indexed unless indexing is explicitly turned off using ``INDEX OFF`` and, separately, values inserted are validated."

2. "The columns are indexed unless indexing is explicitly turned off using ``INDEX OFF`` and unless values inserted are validated."

based on @mfussenegger's response, I take it that the first is the correct interpretation. so how about this:

> Explicitly defined columns within an ``ignored`` object behave the same as those within object columns declared as
``dynamic`` or ``strict`` (e.g., column constraints are still enforced, columns that would be indexed are still indexed unless indexing is explicitly, and so on).